cli/src/print.rs

```diff
@@ -37,14 +37,16 @@ pub fn print_update_message() {
     eprintln!("{:-^50}\n\n", "");
 }
 
-pub fn print_sc_help(app: &mut Command, subcommand: &str) {
-    for sc in app.get_subcommands_mut() {
-        if sc.get_name() == subcommand {
-            let _ = sc.print_help();
-            break;
+pub fn print_sc_help(mut app: &mut Command, subcommands: &[&str]) {
```

Comment:
TIL what `sc` in this function name stands for.... I've breezed past this function many times and never realized that `sc == subcommand`....